pack/scw.json

```
@@ -0,0 +1,380 @@
+		"faction_code": "hero",
+		"hand_size": 6,
+		"health": 10,
+		"hidden": true,
+		"illustrator": "Andrea Di Vitto & Laura Villari",
+		"is_unique": true,
+		"name": "Wanda Maximoff",
+		"pack_code": "scw",
+		"position": 1,
+		"quantity": 1,
+		"recover": 3,
+		"set_code": "scw",
+		"text": "<i>Superpowered Siblings</i> - <b>Action:</b> Discard 2 cards from your hand → draw 2 cards (draw 3 cards instead if Pietro Maximoff is in play). (Limit once per round.)",
+		"traits": "Mystic.",
+		"type_code": "alter_ego"
+	},
+	{
+		"attack": 2,
+		"attack_cost": 1,
+		"card_set_code": "scw",
+		"code": "15002",
+		"cost": 4,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"flavor": "\"I've warned you never to threaten my sister!\"",
+		"health": 4,
+		"illustrator": "Ramon Bachs",
+		"is_unique": true,
+		"name": "Quicksilver",
+		"pack_code": "scw",
+		"position": 2,
+		"quantity": 1,
+		"resource_wild": 1,
+		"set_code": "scw",
+		"set_position": 1,
+		"subname": "Pietro Maximoff",
+		"text": "<b>Action:</b> Ready Quicksilver. (Limit once per phase.)",
+		"thwart": 1,
+		"thwart_cost": 1,
+		"traits": "Avenger.",
+		"type_code": "ally"
+	},
+	{
+		"card_set_code": "scw",
+		"code": "15003",
+		"cost": 0,
+		"deck_limit": 1,
+		"faction_code": "hero",
+		"flavor": "\"I will shape the world as I see fit.\" - Scarlet Witch",
+		"illustrator": "Miguel Sepulveda",
+		"name": "Chaos Magic",
+		"pack_code": "scw",
+		"position": 3,
+		"quantity": 1,
+		"resource_wild": 1,
+		"set_code": "scw",
+		"set_position": 2,
+		"text": "<b>Hero Action:</b> Play a card from your hand, ignoring its resource cost. Discard cards from the top of the encounter deck equal to that card's printed resource cost.",
+		"traits": "Spell. Superpower.",
+		"type_code": "event"
+	},
+	{
+		"card_set_code": "scw",
+		"code": "15004",
+		"cost": 2,
+		"deck_limit": 4,
+		"faction_code": "hero",
+		"illustrator": "Ramon Bachs",
+		"name": "Hex Bolt",
+		"pack_code": "scw",
+		"position": 4,
+		"quantity": 4,
+		"resource_mental": 1,
+		"set_code": "scw",
+		"set_position": 3,
+		"text": "<b>Hero Action</b>: Discard the top 3 cards of the encounter deck. For each card discarded this way that has boost icons equal to:\n- 0, deal 2 damage to an enemy.\n- 1, remove 2 threat from a scheme.\n- 2, draw 1 card.\n- 3+, place a status card on a character.",
+		"traits": "Spell. Superpower.",
+		"type_code": "event"
+	},
+	{
+		"card_set_code": "scw",
+		"code": "15005",
+		"cost": 3,
+		"deck_limit": 3,
+		"faction_code": "hero",
+		"flavor": "\"There is no defense against the Scarlet Witch!\"",
+		"illustrator": "Miguel Sepulveda",
+		"name": "Molecular Decay",
+		"pack_code": "scw",
+		"position": 5,
+		"quantity": 3,
+		"resource_energy": 1,
+		"set_code": "scw",
+		"set_position": 7,
+		"text": "<b>Hero Action</b> <i>attack</i>: Deal 5 damage to an enemy and discard the top 2 cards of the encounter deck. For each boost icon discarded this way, deal 1 additional damage to that enemy.",
```

Just some missing parens:

```suggestion
		"text": "<b>Hero Action</b> <i>(attack)</i>: Deal 5 damage to an enemy and discard the top 2 cards of the encounter deck. For each boost icon discarded this way, deal 1 additional damage to that enemy.",
```